.github/workflows/vb_script.py

```diff
@@ -159,7 +169,11 @@ def update_changelog(path, new_version, pre_release=True):
                         help="True if this is a pre-release version bump, False if it is post release")
     parser.add_argument("--post_release", dest="release_status", action="store_false",
                         help="True if this is a pre-release version bump, False if it is post release")
+    parser.add_argument("--post_pl_release", dest="pl_release_status",
+                        action="store_true", help="True if PL has already been released")
+    parser.add_argument("--pre_pl_release", dest="pl_release_status",
+                        action="store_false", help="False if PL has not been released")
```

Comment:
Added new parameter to track if this version bump is happening before or after a core PL release, and working accordingly

Reply:
As mentioned in the Lightning version bump PR, not sure if we'll need this option.

For now, we have the following release order:

0. Pre-release version bump for PennyLane and all plugins (including PennyLane-Lightning)
1. PennyLane-Lightning release
2. PennyLane release
3. Every other plugin release

As we'd have to change the code if this order changes anyway, maybe it's okay to just leave a comment at the part of the code where it's relevant.